### ISSUE TITLE
Author documentation for Ensemble LDA

### ISSUE THREAD

**Issue by mpenkov** [RaRe-Technologies/gensim#2673]
We've recently renewed our documentation section: https://radimrehurek.com/gensim/auto_examples/index.html

@aloosley @sezanzeb Would you be interested in authoring a new tutorial or howto to cover LDA? If yes, please let me know!

---

Thanks @piskvorky for reviewing our idea and @sezanzeb contribution to gensim.  

In case a audio-visual introduction of eLDA could help with any part of thinking about the code, I wanted to also share this link to a 22 minute eLDA overview presented this year at ML Prague:

https://slideslive.com/38913528/solving-the-text-labeling-challenge-with-ensemblelda-and-active-learning

_Originally posted by @aloosley in https://github.com/RaRe-Technologies/gensim/pull/2282#issuecomment-487158493_

**Comment by aloosley** [comment]
> We've recently renewed our documentation section: https://radimrehurek.com/gensim/auto_examples/index.html. I think the recently added Ensemble LDA feature is great, but are difficult to discover for new users without a top-level document.
> 
> @aloosley @sezanzeb Would you be interested in authoring a new tutorial or howto to cover LDA? If yes, please let me know!

Agreed, this is best way to make the concept and its execution accessible.  Therefore, we're absolutely interested.

I sent Radim a PM asking him to get you in contact with me privately to help me get started (e.g. is there a sphinx-like RST or something else you are using for those tutorials?).  Can you message me on that channel?

**Comment by piskvorky** [comment]
@aloosley in public is preferable – I expect more people will be interested in eLDA. No reason to keep the discussion private. Thanks!

**Comment by aloosley** [comment]
Sounds good.  Anyways I just rediscovered the useful [wiki developer guideline page](https://github.com/RaRe-Technologies/gensim/wiki/Developer-page) so hopefully there is no need for me to ask too many questions after all.

My next step is to take an RST from [docs/src/](https://github.com/RaRe-Technologies/gensim/tree/develop/docs/src) and use it as a template for compiling the requested documentation.  Is there anything else I should know before getting started?

**Comment by mpenkov** [comment]
Yes, please also read https://radimrehurek.com/gensim/auto_examples/howtos/run_doc.html
